packages/stark-ui/assets/styles/_base.scss

```diff
@@ -103,3 +103,23 @@ body {
 .stark-hide {
   display: none;
 }
+
+/* stark-layout */
```

Comment:
Are all this new changes about "stark-layout" actually needed? Because I don't think they are used... and I would say, in case they are used we should maybe just leave it as a specific class in the the Generic Search SCSS instead of having it in the `_base.scss`.